a4747cf70afa929344ed91aec3a4d3dd1db45242 M2: tamper registry build.model_sha256

 registry/chat-param-outcomes-r31.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
commit a4747cf70afa929344ed91aec3a4d3dd1db45242
Author: mut <mut@local>
Date:   Fri Oct 23 20:06:53 2026 -0300

    M2: tamper registry build.model_sha256

diff --git a/registry/chat-param-outcomes-r31.json b/registry/chat-param-outcomes-r31.json
index 61a3d9d..c4baf1c 100644
--- a/registry/chat-param-outcomes-r31.json
+++ b/registry/chat-param-outcomes-r31.json
@@ -3,7 +3,7 @@
  "build": {
   "server_sha256": "b6a31a8e97615a2a1445aeba51e6411e7b02f0a5fa554559347239ee860cd804",
   "library_sha256": "e404b91a81c9ab65756b6a57b97b42115a5f429c5060c4b7f0d717ca595b59ed",
-  "model_sha256": "270cba1bd5109f42d03350f60406024560464db173c0e387d91f0426d3bd256d"
+  "model_sha256": "270cba1bd5109f42d03350f60406024560464db173c0e387d91f0426d3bd256e"
  },
  "base_body": {
   "messages": [
